contents/docs/session-replay/troubleshooting.mdx

```diff
@@ -77,6 +77,12 @@ By default, `base` and `assets` imported from `$app/paths` in Svelte use relativ
 
 For apps with SSR enabled you should set `relative` to `false` in the [`paths` object of your config](https://kit.svelte.dev/docs/configuration#paths). This ensures all asset URLs will correctly be resolved against the base URL rather than the current page.
 
+## Internal users filtering or cohorts-based filters don't work
+
+If you use cohorts to filter sessions, or add cohort condition into the "Filter out internal and test users" setting - it can work incorrectly (or not work at all) while cohorts are calculating (to properly decide what people are in).
```

Comment:
```suggestion
If you use cohorts to filter sessions, or add a cohort condition into the "Filter out internal and test users" setting - it can work incorrectly (or not work at all) while cohorts are calculating.
```